src/menu.c

```diff
@@ -1531,98 +1531,98 @@ void load_menu_text(s32 language) {
-        D_800DFB8C[1] = gMenuText[154];                                     // Insert any Controller Paks
-        D_800DFB8C[2] = gMenuText[155];                                     // you wish to use now!
-        D_800DFB8C[4] = gMenuText[98];                                      // CONTINUE
-        D_800DFBA4[0] = gMenuText[156];                                     // If you wish to use any
-        D_800DFBA4[1] = gMenuText[157];                                     // Rumble Paks then please
-        D_800DFBA4[2] = gMenuText[158];                                     // insert them now.
-        D_800DFBA4[4] = gMenuText[98];                                      // CONTINUE
-        D_800DFBBC[0] = gMenuText[104];                                     // SORRY, BUT YOU
-        D_800DFBBC[1] = gMenuText[105];                                     // CAN'T LOAD GAMES
-        D_800DFBBC[2] = gMenuText[106];                                     // FOR THE SECOND
-        D_800DFBBC[3] = gMenuText[107];                                     // ADVENTURE UNTIL
-        D_800DFBBC[4] = gMenuText[108];                                     // YOU HAVE COMPLETED
-        D_800DFBBC[5] = gMenuText[109];                                     // THE FIRST.
-        D_800DFBBC[7] = gMenuText[98];                                      // CONTINUE
-        gTitleMenuStrings[0] = gMenuText[137];                              // START
-        gTitleMenuStrings[1] = gMenuText[36];                               // OPTIONS
-        gGameSelectTextElemsNoAdv2[0].unk14_a.asciiText = gMenuText[76];    // GAME SELECT
-        gGameSelectTextElemsNoAdv2[1].unk14_a.asciiText = gMenuText[76];    // GAME SELECT
-        gGameSelectTextElemsNoAdv2[3].unk14_a.asciiText = gMenuText[142];   // ADVENTURE
-        gGameSelectTextElemsNoAdv2[5].unk14_a.asciiText = gMenuText[144];   // TRACKS
-        gGameSelectTextElemsWithAdv2[0].unk14_a.asciiText = gMenuText[76];  // GAME SELECT
-        gGameSelectTextElemsWithAdv2[1].unk14_a.asciiText = gMenuText[76];  // GAME SELECT
-        gGameSelectTextElemsWithAdv2[3].unk14_a.asciiText = gMenuText[142]; // ADVENTURE
-        gGameSelectTextElemsWithAdv2[5].unk14_a.asciiText = gMenuText[143]; // ADVENTURE TWO
-        gGameSelectTextElemsWithAdv2[7].unk14_a.asciiText = gMenuText[144]; // TRACKS
-        sBadControllerPakMenuText[0] = gMenuText[69];                       // BAD CONTROLLER PAK
-        sControllerPakFullMenuText[0] = gMenuText[68];                      // CONTROLLER PAK FULL
-        sNoControllerPakMenuText[0] = gMenuText[67];                        // NO CONTROLLER PAK
-        sCorruptDataMenuText[4] = 0;
-        sCorruptDataMenuText[5] = 0;
-        sCorruptDataMenuText[0] = gMenuText[184];                           // CORRUPT DATA.
+            gOptionMenuStrings[1] = gMenuText[ASSET_MENU_TEXT_SUBTITLESOFF];                         // SUBTITLES OFF
```

Comment:
Do you mind wrapping these comments in quotes (e.g. `// "SUBTITLES OFF"`) just so it's more obvious that it represents the string in `gMenuText` and not a reiteration of the `AssetMenuTextIdsEnum`?